src/preact.d.ts

```diff
@@ -141,6 +147,7 @@ declare namespace JSX {
 		y1?:number | string;
 		y2?:number | string;
 		y?:number | string;
+		[propName: string]: any;
```

Comment:
I would guess this is because you are calling `h` directly and not using jsx? if that is the case it would be better to add it in definition of `h` than here